bullet_train/app/views/layouts/docs.html.erb

```diff
@@ -111,6 +111,12 @@
                   <% end %>
                 <% end %>
 
+                <%= render 'account/shared/menu/item', url: '/docs/stylesheets', label: 'Style Sheets' do |p| %>
+                  <% p.icon do %>
+                  <i class="fa-brands fa-js ti ti-brush"></i>
```

Comment:
Not sure why the `<i>` just above and below here are misindented, but the other ones in the file are alright. How about we indent like the rest of them?

```suggestion
                    <i class="fa-brands fa-js ti ti-brush"></i>
```